cellmlmanip/units.py

```diff
@@ -622,3 +635,179 @@ def traverse(self, expr):
 
         raise UnexpectedMathUnitsError(str(expr))
 
+    def convert_expr_recursively(self, expr, to_units):
```

Comment:
Should this maybe be in the `units` module?

Reply:
It it, isn't it? Or do you mean in the `UnitStore` class?

Reply:
I have moved them to the `UnitStore` class now.